src/notificator/notificator_config.py

```
@@ -21,10 +23,32 @@
+    *event_type* is the ``eventTypeNames`` query-parameter value **and** the key
+    used to look up org IDs in the JSON response.
+    """
+
+    application: str
+    event_type: str
+
+
+LIFECYCLE_SUBSCRIPTION = SubscriptionType("life-cycle", "retiring-lifecycle-monthly-report")
+ROADMAP_SUBSCRIPTION = SubscriptionType("roadmap", "roadmap-monthly-report")
+
+DEFAULT_TLS_CERT_PATH = "/tmp/tls/cert.pem"
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/tls/cert.pem"

(S108)